hugo/content/contributing/pull-requests.md

```diff
@@ -7,6 +7,7 @@ weight: 253
 
 So you decided to submit an issue and work it.  Great! Let's get it merged in to the codebase. The following will go a long way to helping get the fix merged in quicker:
 
-1. Create a Pull Request from your Fork to the Develop branch.
-2. Update the checklists in the Pull Request description.
-3. Reference which issues this Pull Request is resolving.
+1. Fork the [Github repository](https://github.com/CrunchyData/crunchy-containers) and make a branch off of the *develop* branch
+2. Create a Pull Request from your Fork back to the *develop* branch.
```

Comment:
I think you can link to the branch using *[develop]*(/CrunchyData/crunchy-containers/tree/develop)

Reply:
Thanks! Done